### DISCUSSION TITLE
How do I get autocomplete to work?

### DISCUSSION THREAD

**Discussion by tulocGH** [zed-industries/zed#10821] [Feature Requests]
Hi there - I have read through the documentation but I can not find any info on how to get autocompletion to work. Also, I have looked through settings. Is auto completion available? Any help would be appreciated

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
